pkg/core/block/block_test.go

```diff
@@ -104,171 +103,175 @@ func TestBlockVerify(t *testing.T) {
-	rawtxBytes, _ := hex.DecodeString(rawtx)
-
-	b := Block{}
-
-	assert.NoError(t, testserdes.DecodeBinary(rawtxBytes, &b))
-	expected := map[string]bool{ // 18 trans
-
-		"009f61f481f47eb7478e887871e4e744669d461b13d68e04250035260171d706": false,
-		"3a62e473c1d67ac561b98e8131f7f7ceded4cd250edb78a6814ec9915930ad93": false,
-		"d56a545d2f9400c09d5aa4e8cc37cc994d5a6892f9c30de95ff69a3b647b27a2": false,
-		"57f6baa9cb23ce9117d66aee7c31ba6d1e82e140a805db1c5992ada849f6a7c7": false,
-		"f700be9c723ed44900ac9d70874b9d8601033bb78883c0a43ab38b5d96c09c11": false,
-		"31674ac8553b371ddf06db6a3aef09b8d6f37da03a8cc2868b71044c54ef0034": false,
-		"44858de48ec97cea2f823128e9d58981dde11f28a6ebf0a2cb745ea13223dd71": false,
-		"317f3ff3768b2aebe3d4866f6e0e8b875cc7937a1b8b5f91be066dc51ed61be2": false,
-		"8c24f44f1533567c71e722f49bc7a4d9b323a09e2950fd975291817578119508": false,
-		"55a7a738aaee8f7e6d7bcd4c8a38813e57763bff8bfb296418b6cac6d5bfb89a": false,
-		"dfa5f84366cf0b48f1b1e9b24a73557e657f6ac21b676528401f5a630aece571": false,
-		"5839fbcbbca68aef41dfa9a371222565519626affad6be0977d38a82259480a5": false,
-		"6873568cae35e4ce0a7d07ef080ef6eb699b2b9dcbc419fad1c4f645ff8579fc": false,
-		"dbb3c0688003bede7e7bc56d2c9d6362b594512ac686820739d963ef91e2eb9e": false,
-		"3d12353cb8bae8be928131580e960a82f37ca3ad6957ad22c8cadc1b21b2dd1a": false,
-		"8ca87fd5843f000939244151ce027bad5c1f30f1867c7054918b7f9a66b949e8": false,
-		"ad088940e45a73e00a3cdb7f3248c67a3f6e5d1f05d4cfd44c4e1f4d26cfef87": false,
-		"908a398dd65dfd2aad6c06090c5a71d5e5280746577a6ddd5a1f2c1453f71ead": false,
-	}
+	//TODO NEO3.0: Update binary
```

Comment:
If you're to comment out whole function like
```
/*
func ... {
}
*/
```
you won't have this massive useless diff.